blacklight.gemspec

```diff
@@ -31,6 +31,9 @@ Gem::Specification.new do |s|
   s.add_dependency "nokogiri",  "~>1.6"     # XML Parser
   s.add_dependency "kaminari", ">= 0.15" # the pagination (page 1,2,3, etc..) of our search results
   s.add_dependency "deprecation"
+  s.add_dependency "elasticsearch-model"
```

Comment:
I wonder if we should be generating these dependencies into downstream apps, instead of requiring them for everyone?